compiler_gym/envs/client_service_compiler_env.py

```diff
@@ -0,0 +1,1372 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
```

Comment:
Needs a module docstring saying that this module defines an implementation of CompilerEnv interface using client/server approach